services/src/main/java/org/exoplatform/news/listener/NewsGamificationIntegrationListener.java

```diff
@@ -57,8 +51,8 @@ public void onEvent(Event<String, String> event) throws Exception {
         Map<String, String> gamificationMap = new HashMap<>();
         gamificationMap.put("ruleTitle", ruleTitle);
         gamificationMap.put("object", news.getUrl());
-        gamificationMap.put("senderId", earnerId); // matches the gamification's earner id
-        gamificationMap.put("receiverId", earnerId);
+        gamificationMap.put("senderId", news.getAuthor()); // matches the gamification's earner id
+        gamificationMap.put("receiverId", news.getAuthor());
```

Comment:
I think we need just the senderId , no need to store the receiverId right ?

Reply:
The gamification object map must contain both the senderId and receiverId params.